sdk/docs/sharable/sdk/quickstart/configure/quickstart-project-structure/02-quickstart-app-business-case.rst

```diff
@@ -4,98 +4,77 @@ The Canton Quickstart example application
 Business case
 -------------
 
-The Canton Quickstart contains an example application that provides both a
-demonstration of a Canton application targeted at production, and a way
-of exercising the supporting developer scaffolding provided by the
-bootstrap.
-
-This example application is a simple license management application that
-allows the application provider to issue licenses to application users.
-Canton Coin is used in this transaction. These users are assumed to be
-retail customers of the provider, with access to Canton and
-a Canton Wallet. We do not assume they are running their any of their own
-infrastructure beyond a (possibly outsourced) validator node.
+The Canton Quickstart contains an example application that provides a demonstration of a Canton application targeted at production, 
```

Comment:
We call the application 'production ready' but let's not say it is 'production.'  That is a very specific statement.